### PR TITLE
New version including account-setup design elements

### PR DESCRIPTION
**Pull request by alexwoll**
New! - redesign input button combo
New! - added locked field, use instead of disabled field
Updated! - changed brand color to green (brand-success color)

skarmklipp 2016-03-04 09 46 40

### CODE REVIEW

File: src/mnd-bootstrap/css/_forms.scss [hunk 1]
```diff
@@ -187,3 +205,23 @@ textarea {
...
+}
+
+.fake-field-locked {
+  width: 100%;
+  color: $text-color-dark;
+  font-size: 16px;
+  padding: 12px 20px;
+  background: $ui-color-lighter;
+  border-radius: 6px;
+  .fa-lock {
```
Comment (mnd-hound) [inline src/mnd-bootstrap/css/_forms.scss]: Rule declaration should be preceded by an empty line

File: src/mnd-bootstrap/css/_forms.scss [hunk 2]
```diff
@@ -187,3 +205,23 @@ textarea {
...
 }
+
+.input-group-btn {
+  .btn {
+    border-radius: 6px;
+  }
+}
+
+.fake-field-locked {
+  width: 100%;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/css/_forms.scss]: Properties should be ordered background, border-radius, color, font-size, padding, width

File: src/mnd-bootstrap/css/_forms.scss [hunk 3]
```diff
@@ -187,3 +205,23 @@ textarea {
...
+  color: $text-color-dark;
+  font-size: 16px;
+  padding: 12px 20px;
+  background: $ui-color-lighter;
+  border-radius: 6px;
+  .fa-lock {
+    padding-top: 3px;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/css/_forms.scss]: Properties should be ordered color, float, padding-top

File: src/mnd-bootstrap/css/_buttons.scss [hunk 4]
```diff
@@ -16,6 +16,10 @@ category: CSS
 <button class="btn btn-default" disabled>Default disabled</button></p>
 <p><button class="btn btn-primary">Primary</button>
 <button class="btn btn-primary" disabled>Primary disabled</button></p>
+<p><button class="btn btn-success">Danger</button>
```
Comment (bzf) [inline src/mnd-bootstrap/css/_buttons.scss]: Maybe this button should say "Success"?

File: src/mnd-bootstrap/css/_forms.scss [hunk 5]
```diff
@@ -187,3 +205,24 @@ textarea {
...
+
+.fake-field-locked {
+  background: $ui-color-lighter;
+  border-radius: 6px;
+  color: $text-color-dark;
+  font-size: 16px;
+  padding: 12px 20px;
+  width: 100%;
+
+  .fa-lock {
```
Comment (bzf) [inline src/mnd-bootstrap/css/_forms.scss]: What do you think about calling this `.lock` instead of coupling it to the Font Awesome namespace? :)

**Comment by bzf** [comment]
Other than the comments, 10/10 would merge! :+1:

File: src/mnd-bootstrap/css/_forms.scss (hunk 5 above)
Comment (alexwoll) [inline src/mnd-bootstrap/css/_forms.scss]: I could, but then I need to add a custom class to the field instead of just using what is already there

**Comment by alexwoll** [comment]
skarmklipp 2016-03-04 10 10 36

File: src/mnd-bootstrap/css/_forms.scss (hunk 5 above)
Comment (bzf) [inline src/mnd-bootstrap/css/_forms.scss]: Gotcha! :+1:

**Pull request merged**